web_poet/testing/__init__.py

```diff
@@ -0,0 +1 @@
+from .utils import INPUT_DIR_NAME, META_FILE_NAME, OUTPUT_FILE_NAME, save_fixture
```

Comment:
Why would users need to access these constants?

I was thinking that the programmatic API we expose for non-pytest users could be the following:

1. save_fixture, to save a fixture
2. load_fixture, to load a fixture :)
3. a function extracted from WebPoetItem.runtest, which actually runs the test

I'm not sure, (2) might not be necessary.

What do you think?

Reply:
I think I thought that `save_fixture` is a convenience function but you could want to write or manipulate the fixture directly. But maybe it's not really needed.

Reply:
If we allow to manipulate fixtures directly, we have less flexibility in changing its structure; it seems to be better not to expose this option. Hm, though it goes against the idea of creating fixtures without any code, just in an editor.

Reply:
File names also aren't enough to know the structure, as there are also folder hierarchies that we could change.

Reply:
Let's try to close it off for now, and discourage "manual" interventions. We can expose it later. In the meantime, we'll have time to iterate on folder and file structure if needed.